x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -85,8 +98,8 @@ export function SparkPlot({
               showLegend={false}
               tooltip="none"
             />
-            <AreaSeries
-              id="area"
+            <SparkelineSeries
```

Comment:
This can't be a LineSeries all the time (or an area series)? What changes?

Reply:
This is necessary because when comparison is enabled/visible we should change the current period sparkline to LineSeries and show the previous period with AreaSeries.

There's no technical problem with changing all sparklines to LineSeries, but all tables that yet don't have comparison implemented would change to a LineSeries, if @formgeist agrees I'm fine too.

Reply:
The thoughts behind the mixed chart types was primarily to create difference in the sparklines between the current and previous time periods. I think if we keep to either line or area, the comparison vanishes a bit because of the size of the chart. @cauemarcondes is it easy to exemplify the two charts (line or area)?

Reply:
To be clear, I am fine with mixing line and area. I was just wondering why it's necessary for the current period to switch, and why it can't always be either a line or an area series.

Reply:
> why it's necessary for the current period to switch, and why it can't always be either a line or an area series.

Because there are some places where the spakline is used and it doesn't have comparison data, like on the Service Inventory page table. If I always show it as a LineSeries it would change this table too, that's why I asked @formgeist .